ccp/src/proof_storage.rs

```diff
@@ -17,30 +17,22 @@
 use std::path::PathBuf;
 use tokio::fs::DirEntry;
 
-use ccp_shared::proof::{CCProof, ProofIdx};
+use ccp_shared::proof::CCProof;
+use ccp_shared::proof::ProofIdx;
 
-const EXPECT_DEFAULT_SERIALIZER: &str = "the default serde serializer shouldn't fail";
 const EXPECT_DEFAULT_DESERIALIZER: &str = "the default serde deserializer shouldn't fail";
 
 #[derive(Debug)]
-pub struct ProofStorageWorker {
+pub(crate) struct ProofStorageDrainer {
```

Comment:
Consumer?

Reply:
Dunno, `drainer` sounds for me also not perfect, but more close to what it does than `consumer`.